src/sentry/rules/conditions/event_attribute.py

```diff
@@ -118,17 +119,23 @@ def _get_attribute_values(self, event: GroupEvent, attr: str) -> Sequence[str]:
             return [getattr(e, path[1]) for e in event.interfaces["exception"].values]
 
         elif path[0] == "error":
-            if path[1] not in ("handled", "unhandled"):
+            if path[1] not in ("handled", "unhandled", "main_thread"):
                 return []
 
-            # Flip "handled" to "unhandled"
-            negate = path[1] == "unhandled"
+            if path[1] in ("handled", "unhandled"):
+                # Flip "handled" to "unhandled"
+                negate = path[1] == "unhandled"
 
-            return [
-                e.mechanism.handled != negate
-                for e in event.interfaces["exception"].values
-                if e.mechanism is not None and getattr(e.mechanism, "handled") is not None
-            ]
+                return [
+                    e.mechanism.handled != negate
+                    for e in event.interfaces["exception"].values
+                    if e.mechanism is not None and getattr(e.mechanism, "handled") is not None
+                ]
+            elif path[1] in ("main_thread"):
+                exception_main_thread = event.data["exception_main_thread"]
```

Comment:
`exception_main_thread` isn't part of `data`, so I likely need to access the field directly? Maybe I need to map this into the `models.py`?

Reply:
I would access the thread data directly, similar to how `handled` and `unhandled` is accessing the exception interface just above.

Reply:
But Snuba does the verification of the thread list comparing to the `thread_id`, why would I duplicate this logic instead of checking the `exception_main_thread` field that does exactly that? I don't get it.

Reply:
@marandaneto I believe this logic is for alert rules, those are a completely different beast and don't necessarily rely on data being indexed in snuba

Reply:
Yeah, it might make sense to break up your PR into two: one for searchable in discover and issues stream, and one to get issue alert rules working with your attribute.